packages/db/postgres/tables.sql

```diff
@@ -25,6 +33,17 @@ CREATE TABLE IF NOT EXISTS public.user
   updated_at      TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now()) NOT NULL
 );
 
+CREATE TABLE IF NOT EXISTS public.user_tag
+(
+  id              BIGSERIAL PRIMARY KEY,
+  user_id         BIGINT                                                        NOT NULL REFERENCES public.user (id),
+  tag             user_tag_type                                                 NOT NULL,
+  -- tag_value is useful for certain tags like STORAGE_LIMIT e.g. tag="STORAGE_LIMIT", tag_value="1TB"
+  tag_value       TEXT                                                                  ,
+  inserted_at TIMESTAMP WITH TIME ZONE DEFAULT timezone('utc'::text, now())     NOT NULL,
+  deleted_at  TIMESTAMP WITH TIME ZONE
```

Comment:
What happens if tag deleted and then created for same user? In other words, should we have `updated_at`?

Reply:
I wonder if it's better to keep it this way to allow to have a "history".
So we would have a new row for every state change, what do you think?

Reply:
From https://www.notion.so/pl-strflt/Enforcing-user-storage-capacity-limits-a29aa59f28484842bea2721f5982234a#c7c52385776f49148c28d4fbdede6f1b we were already keeping a history table, that is basically interacted by admin dashboard. It also has "reason" for blocking, or unblocking. With that in mind, I think we should just have the latest in this table, which is the table that controls the product flows authorization. Any thing that I am missing?

Reply:
I think that is fine, have no strong preference.

But, thinking better about `updated_at` individually, we should still have it as `tag_value` can be updated. Or were you considering deleting and adding a new one?

Reply:
@vasco-santos i believe admin would delete then add a new row to preserve a history.  updated_at muddies history a bit.

Reply:
actually on further thought i'm gonna add `updated_at`.

Reply:
@trigramdev9 what's your thinking behind adding `updated_at`? I thought that the `inserted_at` and `deleted_at` combo to give us history was quite neat. Interested to know what's swayed you the other way.

Reply:
if we want to go with delete and create new one, sounds good to not have `updated_ at`